### ISSUE TITLE
JetBooking: Simple option to require a booking deposit

### ISSUE THREAD

**Issue by BlueteraLLC** [CrocoBlock/suggestions#5476]
In JetBooking, it would be great if there was a simple option to set an amount for a required deposit.  One that can vary per unit rented.  I had to find a way to work around this feature not existing.  I had to create a second Woo product just for the rental deposit, then buy and install the Woo plugin "Force Sell".  That forces the rental deposit to be sold with each booking and allows it to appear as an itemized item in confirmation emails and checkout.  I would rather not have to pay for another plugin to make this work.

I did experiment with creating a meta field for the rental deposit and then adding it in the JetEngine form calculations.  That worked, but it would not itemize the rental amount in checkout and confirmation emails/order receipts.  So I decided not to use that.

Or, if it cannot be set per unit, at least allow us to enable and set deposit amounts in global settings.  Per rental/unit would be better though.

**Comment by Gareth-Find-A-New-Website** [comment]
Hiya, 

I used this plugin to allow deposits Woo Deposits and Partial Payments - I've opened a ticket to see if croco can work with it as jetbooking won't allow me to disable the deposit option if arrival is within 30 days. 

Happy to send you a zip of the plugin to see if it meets your needs before you buy it. Then hopefully croco will make them work together :o)

**Comment by asteriskoit** [comment]
Yes, please. This is a very much needed feature.

**Comment by loran750** [comment]
same !

**Comment by evojaime** [comment]
It would be extremely useful to have a native deposit / partial payment option in JetBooking, especially when it’s used together with WooCommerce.

Right now, JetBooking assumes that the customer will always pay 100% of the booking amount upfront. For many businesses (campgrounds, rentals, accommodations, events, etc.), this is not how their model works in real life. They usually:

- Charge a deposit (fixed amount or percentage) at the moment of booking
- Collect the remaining balance later (on arrival, a few days before the booking, or via a second payment link)

Without a built-in deposit feature, we have to use complicated workarounds:

- Creating separate “deposit” products in WooCommerce
- Manually adjusting booking prices or statuses in the backend
- Handling the remaining balance completely outside of JetBooking / WooCommerce

These workarounds create a lot of friction, increase the risk of errors, and make the system much harder to use for clients who are not technical.

A proper implementation inside JetBooking could look like:

- Ability to set a deposit type: fixed amount or % of total
- Ability to choose when the remaining balance is due (manual tracking is fine at first; automation could come later)
- Correct sync with WooCommerce orders and JetBooking prices, so reporting and accounting stay accurate

Having deposit payments supported out of the box would:

- Reduce cancellations (people commit more easily to a deposit than to paying 100% upfront)
- Make JetBooking much more usable for real-world booking businesses (especially in tourism, camping, rentals, etc.)
- Avoid fragile custom code and third-party hacks

In short: deposit payments are not a “nice to have”, they’re a core requirement for a lot of businesses using JetBooking. A native solution would add a lot of value to the plugin.

**Comment by asteriskoit** [comment]
+1
